.github/workflows/build.yml

```diff
@@ -26,7 +26,7 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-name: Build
```

Comment:
What would happen to the ![Build](https://github.com/openthread/openthread/workflows/Build/badge.svg) badge in README.md?